apps/meteor/app/lib/server/lib/notifyListener.ts

```diff
@@ -501,3 +503,263 @@ export async function notifyOnUserChangeById({ clientAction, id }: { id: IUser['
 
 	void notifyOnUserChange({ id, clientAction, data: user });
 }
+
+export async function notifyOnSubscriptionChangedByUserAndRoomId(
```

Comment:
any reason you chose to name this function like `ByUserAndRoomId`? the reason I ask is because usually (if not always) we use "room" first.. so this decision makes code look like this:
```ts
	const { modifiedCount } = await Subscriptions.hideByRoomIdAndUserId(rid, userId);
	if (modifiedCount) {
		void notifyOnSubscriptionChangedByUserAndRoomId(userId, rid);
	}
```

we can see the parameters are swiped.. they're correct, but I wonder if it would be better to have them in the same "position".. it would make it easier to look and prone to copy&paste bugs..